tests/MockServer.js

```diff
@@ -1,6 +1,8 @@
 /*!
- * Copyright (c) 2019-2020 Digital Bazaar, Inc. All rights reserved.
+ * Copyright (c) 2018-2019 Digital Bazaar, Inc. All rights reserved.
```

Comment:
Why are copyright dates being rolled back?

Reply:
Must have been from an older commit. I've updated it.